### PR TITLE
added bitbucket ci config. alligned feature indenting

### PR DESCRIPTION
**Pull request by holdenmcg**
Added a Bitbucket CI snippet to the docs. 

image

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-ci/sample-ci-configs.md [hunk 1]
```diff
@@ -296,6 +296,40 @@ workflows:
...
+## Bitbucket
+
+```yaml
+image: atlassian/default-image:latest
+
+pipelines:
+  default:
+    - parallel:
+      - step:
+          name: 'Run Scan with current branch'
```
Comment (pabloest) [inline docs/semgrep-ci/sample-ci-configs.md:308]:
```suggestion
          name: 'Run Semgrep scan with current branch'
```

File: docs/semgrep-ci/sample-ci-configs.md [hunk 2]
```diff
@@ -296,6 +296,40 @@ workflows:
...
+            # - export SEMGREP_REPO_NAME=$BITBUCKET_REPO_FULL_NAME
+            # - export SEMGREP_BRANCH=$BITBUCKET_BRANCH
+            # - export SEMGREP_JOB_URL="${SEMGREP_REPO_URL}/addon/pipelines/home#!/results/${BITBUCKET_PIPELINE_UUID}"
+            # - export SEMGREP_COMMIT=$BITBUCKET_COMMIT
+            # - export SEMGREP_PR_ID=$BITBUCKET_PR_ID
+            # - export $SEMGREP_APP_TOKEN
+            - semgrep ci --config auto
+```
+
+### Feature Support
```
Comment (pabloest) [inline docs/semgrep-ci/sample-ci-configs.md:323]:
```suggestion
### Feature support
```

**Review by pabloest — APPROVED** [review]
Approved with some minor nits / comments

File: docs/semgrep-ci/sample-ci-configs.md (hunk 1 above)
Comment (holdenmcg) [reply docs/semgrep-ci/sample-ci-configs.md]: Added 👍

File: docs/semgrep-ci/sample-ci-configs.md (hunk 2 above)
Comment (holdenmcg) [reply docs/semgrep-ci/sample-ci-configs.md]: Fixed! 👍

**Pull request merged**